### PR TITLE
[el] Create GreekRedundancyRule (#5209)

### PR DESCRIPTION
**Pull request by giorgossideris**
**Example**
image

*You can see more about redundancy [here](https://en.wikipedia.org/wiki/Redundancy_(linguistics)).*

### CODE REVIEW

File: languagetool-language-modules/el/src/test/java/org/languagetool/rules/el/GreekRedundancyRuleTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package org.languagetool.rules.el;
```
Comment (danielnaber) [inline languagetool-language-modules/el/src/test/java/org/languagetool/rules/el/GreekRedundancyRuleTest.java:1]: Could you add the standard license header here, too?

**Comment by danielnaber** [comment]
Thanks! Have you locally tested this with a large corpus, to make sure the matches make sense in all/most contexts?

**Comment by giorgossideris** [comment]
> 
> 
> Thanks! Have you locally tested this with a large corpus, to make sure the matches make sense in all/most contexts?
@danielnaber  thank you for the review! Yes I tested it with the wikipedia corpus and no matches were found, that I believe is logical, as carefully written text ought not to contain redundancies (obviously no false positives were detected that is good).

File: languagetool-language-modules/el/src/test/java/org/languagetool/rules/el/GreekRedundancyRuleTest.java (hunk 1 above)
Comment (giorgossideris) [reply languagetool-language-modules/el/src/test/java/org/languagetool/rules/el/GreekRedundancyRuleTest.java:19]: Done!

**Comment by danielnaber** [comment]
@giorgossideris Maybe you could also test with https://tatoeba.org, just to be sure? Unlike Wikipedia, it also contains colloquial texts.

**Comment by giorgossideris** [comment]
> 
> 
> @giorgossideris Maybe you could also test with https://tatoeba.org, just to be sure? Unlike Wikipedia, it also contains colloquial texts.

Of course, I will do it now.

**Comment by giorgossideris** [comment]
I tested with the sentences and detailed sentences from https://tatoeba.org/en/downloads and no matches were found neither.

**Pull request merged**